src/Toolkit/Toolkit.iOS/UI/Controls/PopupViewer/PopupViewer.iOS.cs

```diff
@@ -243,7 +245,7 @@ private void Refresh()
             if (!string.IsNullOrWhiteSpace(PopupManager.CustomDescriptionHtml))
```

Comment:
Can we remove the null-check on `_detailsList` above, since it's a nonnull field?